src/fides/api/ctl/migrations/versions/ff782b0dc07e_add_replaceable_field_to_custom_connector_template.py

```diff
@@ -0,0 +1,28 @@
+"""add replaceable field to custom_connector_template
+
+Revision ID: ff782b0dc07e
+Revises: 6d6b0b7cbb36
+Create Date: 2023-03-29 23:41:26.164600
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "ff782b0dc07e"
+down_revision = "6d6b0b7cbb36"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "custom_connector_template",
+        sa.Column(
+            "replaceable", sa.Boolean(), unique=False, nullable=False, default=False
```

Comment:
Adding a `replaceable` flag on the custom connector template to denote if this custom connector should be replaced once a newer version of the same connector type is bundled with Fides. This is to support the use case where we give a customer a patch in between releases in the form of a connector template zip file

Reply:
Open to other names!